### PR TITLE
Page profile edit

### PR DESCRIPTION
**Pull request by kw222mi**

### CODE REVIEW

File: traino/app/profile/[slug]/edit/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import React, { useState, useRef } from "react";
+import { useTrainoContext } from "@/app/components/Context";
+import { isValidEmail, ifAliasExist } from "../../../components/functions";
```
Comment (SofiaHjerpe) [inline traino/app/profile/[slug]/edit/Modal.jsx:3]: Well done! is it posible to use the same component for all modals? And use props to see witch one it is?

File: traino/app/profile/[slug]/edit/ModalEducation.jsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  const handleAdd = () => {
+    {
+      /** 
+      const educationObject = {
+        education: education,
+        educationSpec: educationSpec,
+        startdate: startdate,
+        stopdate: stopdate,
+      };
+      console.log("Saving education object:", educationObject);
```
Comment (SofiaHjerpe) [inline traino/app/profile/[slug]/edit/ModalEducation.jsx:51]: Can you remove this comments?

**Review by SofiaHjerpe — COMMENTED** [review]
Very good! I just wonder if it is possible to merge modals into one?

File: traino/app/profile/[slug]/edit/ModalEducation.jsx (hunk 2 above)
Comment (kw222mi) [reply traino/app/profile/[slug]/edit/ModalEducation.jsx:51]: I fix that!

File: traino/app/profile/[slug]/edit/Modal.jsx (hunk 1 above)
Comment (kw222mi) [reply traino/app/profile/[slug]/edit/Modal.jsx:3]: "It's not possible to reuse the same approach for all data without it becoming overly complicated. However, I have reused it whenever possible.

**Review by SofiaHjerpe — APPROVED** [review]

**Pull request merged**